storage/tianmu/core/engine.h

```diff
@@ -313,6 +313,7 @@ class ResultExportSender final : public ResultSender {
 };
 
 enum class tianmu_var_name {
+  TIANMU_TIMEOUT,
```

Comment:
The first enum item should be set a default value.

Reply:
OK, I will fix it;